_kb/liquidity-tiers.md

```diff
@@ -0,0 +1,37 @@
+---
+title: NuBits Liquidity Guide
+nav_heading: NuBits
+filename: liquidity-tiers.md
+permalink: /nubits-liquidity/
+layout: kb
+lang: en
+callouts:
+---
+
+A motion was put forth by Jordan Lee on [Nov 12th 2014](https://discuss.nubits.com/t/finalized-evolution-of-liquidity-operations/618) to help guide the development of liquidity operations for the Nu Network. The design calls for recognizing six tiers to the liquidity operations that support the NuBits $1 peg.
+
+## NuBits Liquidity Tiers
+
+### Tier 1
+
+This liquidity is immediately available, being on the order book. It is used by NuBot to maitain the 1$ peg on exchanges.
+
+### Tier 2
+
+This liquidity sits on exchanges but is not placed on order books and can be promoted to the order book (Tier 1) in a couple seconds.
```

Comment:
Ok, here we need some thoughts on tier1 and tier2.

Tier1 is on the orderbook AT the best price (1$ +- fees and spread) . This is "the wall" .
Tier2 can be sitting on the account OR placed on the orderbook at premium prices .